js-miniapp-bridge/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "js-miniapp-bridge",
-  "version": "0.1.0",
+  "version": "1.1.0-alpha",
```

Comment:
This one doesn't actually get released since it's manually integrated into the SDK, so the version number here doesn't affect anything. But I guess it's still good to keep them on the same version.

Reply:
Exactly for the sync :)